crates/bevy_ecs/src/storage/mod.rs

```diff
@@ -41,3 +41,10 @@ pub struct Storages {
     /// Backing storage for `!Send` resources.
     pub non_send_resources: Resources<false>,
 }
+
+impl Storages {
+    pub fn shrink_to_fit(&mut self) {
```

Comment:
This should probably be `pub(crate)`. There's no way for a user of `bevy_ecs` to get a `&mut Storages` without being unsound.